File: executable/r/simple-subplot.r
library(plotly)
p <- plotly(username='test-runner', key='9h29fe3l0x')

trace1 <- list(
  x = c(1, 2, 3), 
  y = c(4, 5, 6)
)
trace2 <- list(
  x = c(20, 30, 40), 
  y = c(50, 60, 70), 
  xaxis = "x2", 
  yaxis = "y2"
)

layout <- list(
  xaxis = list(domain = c(0, 0.45)), 
  xaxis2 = list(domain = c(0.55, 1)), 
  yaxis2 = list(anchor = "x2")
)

response <- p$plotly(trace0, trace1, kwargs=list(layout=layout, filename="simple-subplot", fileopt="overwrite"))
url <- response$url
filename <- response$filename

File: auto-docs/executables/node_js/simple_subplot.js
// Learn about API authentication here: plot.ly/nodejs/getting-started
// Find your api_key here: plot.ly/settings/api

var plotly = require('plotly')('TestBot', 'r1neazxo9w')


var trace1 = {
  x: [1, 2, 3], 
  y: [4, 5, 6], 
  type: "scatter"
};
var trace2 = {
  x: [20, 30, 40], 
  y: [50, 60, 70], 
  xaxis: "x2", 
  yaxis: "y2", 
  type: "scatter"
};
var data = [trace1, trace2];
var layout = {
  xaxis: {domain: [0, 0.45]}, 
  yaxis2: {anchor: "x2"}, 
  xaxis2: {domain: [0.55, 1]}
};
var graphOptions = {layout: layout, filename: "simple-subplot", fileopt: "overwrite"};
plotly.plot(data, graphOptions, function (err, msg) {
    console.log(msg);
});
